components/atoms/CriteriaBox.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from "prop-types";
+
+export function CriteriaBox(props) {
+  return (
+    <div className="block flex-1 my-1 mx-1 p-4 md:my-0 md:mx-0 rounded-lg shadow-lg border border-gray-200 ">
+      {/* <label
+        className="text-gray-700 font-display font-bold"
+        data-cy={props.dataCy}
+      >
+        {props.criteriaTitle}
+      </label> */}
```

Comment:
Please remove the commented out `<label>`

Reply:
oops, I took this out then added it back to see where it was breaking. Forgot to take it back out again. It's gone now.